src/main/cpp/systemds.cpp

```diff
@@ -54,9 +54,9 @@
 #define RELEASE_ARRAY(env, input, inputPtr, numThreads) \
 	env->ReleasePrimitiveArrayCritical(input, inputPtr, 0)
 // ( maxThreads != -1 && ((int)numThreads) == maxThreads ? env->ReleasePrimitiveArrayCritical(input, inputPtr, 0) :  env->ReleaseDoubleArrayElements(input, inputPtr, 0) )
-  
+
 // -------------------------------------------------------------------
-JNIEXPORT void JNICALL Java_org_apache_sysds_utils_NativeHelper_setMaxNumThreads
+/*JNIEXPORT void JNICALL Java_org_apache_sysds_utils_NativeHelper_setMaxNumThreads
```

Comment:
I understand that you could not make it work without commenting it out. 

But can we do a second try? or in the PR at least not comment it out.

One alternative strategy is to compile a separate file  systemds_img.cpp ? maybe.

Reply:
on it !